class AuthenticationException(Exception):
    """Raised when the authentication fail"""

    pass
